recipes-qt/automotive/qtapplicationmanager_git.bbappend

```diff
@@ -0,0 +1,30 @@
+#SRC_URI += "https://github.com/qt/qtapplicationmanager.git;branch=5.8"
+#SRCREV = "23a0e5a929b5b03f91ae532a765e2b7a543c3a99"
+
+RDEPENDS_${PN} += "${PN}-softwarecontainer" 
+
+EXTRA_QMAKEVARS_PRE += "\
+    -config enable-examples \
+"
+
+do_install_append(){
+    install -d ${D}${libdir}	
+    install -m 755 ${B}/examples/softwarecontainer-plugin/libsoftwarecontainer-plugin.so ${D}/usr/lib/
+    echo '
+containers:
+    selection:
+    - com.pelagicore.radio: "softwarecontainer"
+    - "*": "process"
+
+    softwarecontainer:
+        dbus: system
+
+plugins:    
+    container: [ "/usr/lib/libsoftwarecontainer-plugin.so" ]' >> ${D}/opt/am/config.yaml
```

Comment:
Wouldn't it make more sense to create a new softwarecontainer.yaml file and add "-c softwarecontainer.yaml" to the commandline of appman ?